src/transcoder/transcodedialog.ui

```diff
@@ -57,6 +57,11 @@
           <string>Directory</string>
          </property>
         </column>
+        <column>
+         <property name="text">
+          <string>Import Directory</string>
+         </property>
+        </column>
```

Comment:
It's not necessary to show this information to the user - this can be removed if desired.